pkgs/development/compilers/opensmalltalk-vm-squeak/default.nix

```
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchurl, makeWrapper, alsa-lib, libGL, libpulseaudio, libuuid
+, xorg, ... }:
```

```suggestion
, xorg }:
```

That isn't used, right?